zephyr/rz/rz_cfg/fsp_cfg/bsp/rzg3s/bsp_clock_cfg.h

```diff
@@ -0,0 +1,58 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#ifndef BSP_CLOCK_CFG_H_
+#define BSP_CLOCK_CFG_H_
+#define BSP_CFG_CLOCKS_SECURE (0)
+#define BSP_CFG_CLOCKS_OVERRIDE (0)
+#define BSP_CFG_CLOCK_OSCCLK_HZ (24000000) /* OSC 24000000Hz */
```

Comment:
@binhnguyen2434 : May I confirm the reason of fixed clock here but not set in device tree?

Reply:
@thaoluonguw : Yes, it will be set by device tree, but it and other clock configurations are being updated by another branch for clock control support. In this branch, I would like to hard code it.